src/modules/emaiMethods.ts

```diff
@@ -1,11 +1,17 @@
+import { ICalCalendarMethod } from 'ical-generator';
+import RRule from 'rrule';
```

Comment:
Default imports from rrule for some reason are always broken on my machine. Only `import { Rrule }` works